CogNative/main.py

```diff
@@ -17,6 +17,19 @@
 import sys
 args = sys.argv[1:]
 
+# -help flag displays help instead of processing anything
+if '-help' in args:
+    print("""
+CogNative CLI FLags: 
+    -sampleAudio <PATH>: audio file of voice to copy
```

Comment:
```suggestion
    -sampleAudio <PATH>: audio file of voice to clone
```

Maybe this change to further distance ```-sampleAudio``` from ```-dialogueAudio```.